### DISCUSSION TITLE
OneDrive it self don't sync anymore after adding a Sharepoint Folder

### DISCUSSION THREAD

**Discussion by pepperbilly** [abraunegg/onedrive#1461] [Q&A] [answered]
Hi

After adding a SharePoint Folder in ~/.config/onedrive/config

sync_dir="~/Local_Sharepointfolder" drive_id="thedriveid"

My default added OneDrive Folder is not syncing anymore. Only the SharePoint Folder will sync.
As soon as i remove the Sharepoint Folder from the config File and run

onedrive --synchronize --resync

the OneDrive Folder will sync again.

Any Idea of what i did wrong?

thanks for your Help

Sascha

Ubuntu 20.04.2 LTS
Release: 20.04

**Comment by abraunegg** [comment, marked as answer]
@pepperbilly 
For each SharePoint Library that you wish to sync, you need a unique configuration for each entry.

This is why your default folder does not sync with your single configuration - because you have configured it for SharePoint. To sync both your OneDrive & SharePoint you will need 2 configurations of the client. Refer to:
* https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md
* https://github.com/abraunegg/onedrive/blob/master/docs/SharePoint-Shared-Libraries.md


Also, as you are using Ubuntu, make sure you are building from source and follow when releases are done. The Ubuntu client is out of date and is not supported.
